src/client/src/Client.c

```diff
@@ -880,6 +880,46 @@ STATUS kinesisVideoStreamFormatChanged(STREAM_HANDLE streamHandle, UINT32 codecP
     return retStatus;
 }
 
+/**
+ * Update NALu adaptation flags
+ *
+ * NOTE: The operation is permitted in non-streaming states
+ */
+STATUS kinesisVideoStreamSetNalAdaptionFlags(STREAM_HANDLE streamHandle, UINT32 nalAdaptationFlags)

```

Comment:
this one too `kinesisVideoStreamSetNalAdaptionFlags` --> `kinesisVideoStreamSetNalAdaptationFlags`